asv_bench/benchmarks/join_merge.py

```diff
@@ -444,9 +349,12 @@ def setup(self):
         self.ts1 = Series(np.random.randn(self.sz), self.idx1)
         self.ts2 = Series(np.random.randn(self.sz), self.idx2)
 
-    def time_series_align_left_monotonic(self):
-        self.ts1.align(self.ts2, join='left')
-
     def sample(self, values, k):
         self.sampler = np.random.permutation(len(values))
         return values.take(self.sampler[:k])
+
```

Comment:
Ah, yes, I probably did the cleanup of this file before that (or I didn't update), will cleanup the merge asof benches as well then (they are now in `merge_asof_noby`, `merge_asof_byobject` and `merge_asof_byint` classes